src/main/kotlin/org/jetbrains/research/testspark/actions/controllers/TestGenerationController.kt

```diff
@@ -26,6 +27,7 @@ class TestGenerationController {
     private fun showGenerationRunningNotification(project: Project) {
         val terminateButton: AnAction = object : AnAction("Terminate") {
             override fun actionPerformed(e: AnActionEvent) {
+                indicator?.stop()
```

Comment:
It would be nice to close this popup modal after a user clicks "Terminate." Otherwise, I can click as many times as I like (it does not cause errors, though).